posts/20230622/index.qmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Let’s Verify Step by Step"
+author:
+    - Akshata Mohan
+    - Hongsup Shin
+date: "2023-06-22"
+image: "fig1.png"
+description: "TBD"
+categories: [paper]
+---
+
+:::{.callout-note}
+- Paper: [*Let’s Verify Step by Step*](https://arxiv.org/abs/2305.20050)
+- Presenter: [Akshata](https://www.linkedin.com/in/akshata-mohan/)
+- Attendees: [Hongsup](https://www.linkedin.com/in/hongsupshin/), [Meghann](https://www.linkedin.com/in/meghann-agarwal/), [Saina](https://www.linkedin.com/in/saina199/)
+:::
+
+## Why this paper?
+
+## Paper summary
```

Comment:
The paper evaluates different approaches to solving a dataset comprising of math problems. With this approach, they trained the model to get the right answer and but also "think" through the problem to arrive at the right answer. There are numerous benefits to this approach:
1. Model reasoning is explainable as each step of the process is provided a probability score (positive, negative and neutral)
2. Wrong steps that would lead to incorrect outcomes are identified early on in the training process
3. Wrong steps that would lead to the correct solution are also identified and discouraged 
4. Model could be trained to reason and model its behaviors that matches human values which leads to reduction in hallucinations, false information and potentially dangerous outcomes.

The benefits of this process based approach are multi-fold:  reduction in noise, human thinking alignment and explainable models make this a very interesting and exciting approach.